core/domain/exp_services.py

```diff
@@ -1935,14 +2013,16 @@ def validate_exploration_for_story(
     return validation_error_messages
 
 
-def update_exploration(
+def compute_models_for_updating_exploration(
     committer_id: str,
     exploration_id: str,
     change_list: Optional[List[exp_domain.ExplorationChange]],
     commit_message: Optional[str],
     is_suggestion: bool = False,
-    is_by_voice_artist: bool = False
-) -> None:
+    is_by_voice_artist: bool = False,
+    is_synchronous: bool = False,
+    should_put_models: bool = True
```

Comment:
Don't have this as an option. Modifying the datastore is a dangerous enough operation that this should not be done as an argument to a general thing ... much less one that has a default value which callers can therefore easily miss.